### PR TITLE
Required Changes

### PR DESCRIPTION
**Pull request by deniodev**
Hi @souad988 ,

Good day.

Please find below the changes that I made.

- Add a h1 title on HTML file;
- Add a page title on the HTML file;
- Add a valid license file and update the link to it in the README file;
- Move the README file into the features branch.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <h1>Hello Microverse!</h1>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    
+</body>
```
Comment (souad988) [inline index.html]: Kindly add a h1 title to your page as requested in the requirements

File: index.html [hunk 2]
```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
```
Comment (souad988) [inline index.html]: Please add a valid title to your page referring to your project exp: Hello Microverse

**Review by souad988 — CHANGES_REQUESTED** [review]
Hi @deniodev , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

212385281-4797c58a-d358-423a-aa7c-832c22203865


## Highlights
✔No linter errors 
✔Great efforts

## Required Changes ♻️
_Check the comments under the review._
_ Also consider adding a valid licence file and update the link to it in your README this is required for legal reasons
_also put your README file in this branch as I can not give feedback about it now  and always do all the work from your feature branch submit for review and only when approved that you can move your code to the main branch 
_ Kindly change this PR title and summary to something that describes better the work done in this feature  otherwise this title is misleading and it doesn't give any idea about the work and the feature.
_Last thing is Please make all these changes in this same branch and submit for review .
_ you have done great keep up the hard work 💪💪💪
 
## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: .github/workflows/linters.yml [hunk 3]
```diff
@@ -0,0 +1,56 @@
+name: Linters
+
```
Comment (adel-gu) [inline .github/workflows/linters.yml:2]: - [OPTIONAL] In future projects consider giving your branch meaningful naming that conveys an action or a feature that you are trying to implement 💪🏼 

eg:
1. `hello-microvers`
2.  `setup-hello-microverse`

**Review by adel-gu — APPROVED** [review]
# Status: Approved 🟢 

Hi @deniodev 👋🏼 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- You added the H1 heading ✔️ 
- The app has a meaningful title now ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: .github/workflows/linters.yml (hunk 3 above)
Comment (deniodev) [reply .github/workflows/linters.yml:2]: Thank you for your time.

**Pull request merged**